specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromIds.tsx

```diff
@@ -222,9 +222,9 @@ export function RecordSelectorFromIds<SCHEMA extends AnySchema>({
                   className={`flex-1 ${dialog === false ? '-ml-2' : '-ml-4'}`}
                 />
               )}
-              {hasAttachments && (
+              {hasAttachments && title !== undefined ? (
```

Comment:
you can make the code more self-documenting by defining a variable like `const isNewRecordSet = title === undefined` and then using it here.

and while you are at it, improve your condition by adding isInRecordSet to it:

```ts
const isNewRecordSet = isInRecordSet && title === undefined;
```